packages/mobile/src/invite/saga.ts

```diff
@@ -346,25 +350,13 @@ export function* skipInvite() {
 function* addTempAccountToWallet(inviteCode: string) {
   Logger.debug(TAG + '@addTempAccountToWallet', 'Attempting to add temp wallet')
   try {
-    const contractKit = yield call(getContractKit)
-    let tempAccount: string | null = null
-    const fornoMode = yield select(fornoSelector)
-    if (fornoMode) {
-      tempAccount = contractKit.web3.eth.accounts.privateKeyToAccount(inviteCode).address
-      Logger.debug(
-        TAG + '@redeemInviteCode',
-        'web3 is connected:',
-        String(yield call(contractKit.web3.eth.net.isListening))
-      )
-      addLocalAccount(inviteCode)
-    } else {
-      // Import account into the local geth node
-      tempAccount = yield call(
-        contractKit.web3.eth.personal.importRawKey,
-        trimLeading0x(inviteCode),
-        TEMP_PW
-      )
-    }
+    yield call(waitForGethConnectivity)
+    // Import account into the local geth node
+    const tempAccount: string | null = yield call(
```

Comment:
if this is possible null we should probably throw, no?

Reply:
the `addAccount` will throw and never return null